pandas/tests/series/methods/test_equals.py

```diff
@@ -31,9 +31,8 @@ def test_equals_list_array():
     s2 = s1.copy()
     assert s1.equals(s2)
 
-    # TODO: Series equals should also work between single value and list
-    # s1[1] = 9
-    # assert not s1.equals(s2)
+    s1[1] = 9
+    assert not s1.equals(s2)
```

Comment:
can you add all of the missing cases
e.q. `not s1.equals('foo')`

also we have a bunch of tests for array_equivalent, these need a battery of tests

Reply:
parameterized values in test_equals and added tests for array_equivalent with a Series